assets/javascripts/discourse/routes/notebook.js.es6

```diff
@@ -1,9 +1,10 @@
 /**
  * Route for the path `/notebook` as defined in `../notebook-route-map.js.es6`.
  */
-export default Discourse.Route.extend({
+import DiscourseRoute from "discourse/routes/discourse";
+export default DiscourseRoute.extend({
   renderTemplate() {
     // Renders the template `../templates/notebook.hbs`
-    this.render('notebook');
-  }
+    this.render("notebook");
+  },
```

Comment:
**Suggestion**: To keep this file’s code style in line with the rest of JavaScript files in this project, I’d like to make the following changes:

```suggestion
    this.render('notebook');
  }
```